src/js/components/sharedComponents/header/Header.jsx

```diff
@@ -140,7 +140,7 @@ export default class Header extends React.Component {
                                 alt="U.S. flag" />
                         </div>
                     </div>
-                    {infoBanner}
+                    
```

Comment:
we can leave this here, it will be null. We want to remove the banner w/o removing the functionality and allow the next developer who needs to implement a banner to be able to do so pretty easily.

I would probably rename `openCovidModalContainer` to something more generic and readable that  would make sense for any kind of banner.